src/scripts/stylelint/.stylelintrc.js

```diff
@@ -2,8 +2,9 @@ module.exports = {
   // You can extend existing configurations
   // See https://stylelint.io/user-guide/configure#extends
   extends: [
+    "stylelint-config-property-sort-order-smacss",
```

Comment:
Move this to a different PR and mark it as a feat that updates the stylelinting rules for apps.